ts/src/coinex.ts

```diff
@@ -327,35 +328,64 @@ export default class coinex extends Exchange {
             const currencyId = this.safeString (currency, 'asset');
             const networkId = this.safeString (currency, 'chain');
             const code = this.safeCurrencyCode (currencyId);
-            const precision = this.parseNumber (this.parsePrecision (this.safeString (currency, 'withdrawal_precision')));
+            const precisionString = this.parsePrecision (this.safeString (currency, 'withdrawal_precision'));
+            const precision = this.parseNumber (precisionString);
+            const canDeposit = this.safeValue (currency, 'can_deposit');
+            const canWithdraw = this.safeValue (currency, 'can_withdraw');
+            const feeString = this.safeString (currency, 'withdraw_tx_fee');
+            const fee = this.parseNumber (feeString);
+            const minNetworkDepositString = this.safeString (currency, 'deposit_least_amount');
+            const minNetworkDeposit = this.parseNumber (minNetworkDepositString);
+            const minNetworkWithdrawString = this.safeString (currency, 'withdraw_least_amount');
+            const minNetworkWithdraw = this.parseNumber (minNetworkWithdrawString);
             if (this.safeValue (result, code) === undefined) {
                 result[code] = {
                     'id': currencyId,
                     'numericId': undefined,
                     'code': code,
-                    'info': currency,
+                    'info': [],
```

Comment:
why removing the raw object from info?

Reply:
@carlosmiei I push raw object each time in row 420
`result[code]['info'].push (currency);`
`Info` is now an array, so not to duplicate 1st record in `info` I need to leave it empty and push raw object afterwards.

Reply:
@ProfHollander sorry I still don't get the "so not to duplicate 1st record in info I need to leave it empty and push raw object afterwards." part, can you elaborate a bit more?

Reply:
@carlosmiei 
In first iteration when `result[code]` id not defined I leave `info` as empty array and afterwards, when I add `networks` I also return raw object to `info`.
If I won't leave it empty at first I will duplicate 1st entry.
I will rework code in a few minutes a little bit, maybe it will be more convinient for reading and understanding.

Reply:
@ProfHollander I would appreciate that because right now I'm not convinced `info` should be an array here

Reply:
@carlosmiei now I understand what confuses you. I also can't say that array is the best option here.
Coinex don't give in API common info about currency.  They response info about each network:
```
"USDT-TRC20": {
"asset": "USDT",
 "chain": "TRC20",
 "withdrawal_precision": 6,
 "can_deposit": true,
 "can_withdraw": true,
 "deposit_least_amount": "1.2",
 "withdraw_least_amount": "1.2",
 "withdraw_tx_fee": "1.2",
 "explorer_asset_url": "https://tronscan.org/#/token20/TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t"
 },
```
 
```
 "USDT-ERC20": {
 "asset": "USDT",
 "chain": "ERC20",
 "withdrawal_precision": 6,
 "can_deposit": true,
 "can_withdraw": true,
 "deposit_least_amount": "7",
 "withdraw_least_amount": "7",
 "withdraw_tx_fee": "7",
 "explorer_asset_url": "https://etherscan.io/token/0xdac17f958d2ee523a2206206994597c13d831ec7"
 },
```
 ....
 In current version when we add `info` we add only info about first `network` that we get in API response. (now it's only TRC20 network).
 
 I added all networks to `result[code]['info']`, and also added the same `info` to each `network`.
 
 I think that in `result[code]['info']` it's better to leave rawObject from all networks. It's better than only rawObject from only first network.